common/trajectory.cpp

```diff
@@ -10,6 +10,7 @@
 #include <algorithm>
 #include "regexpatterns.hpp"
 #include "trajectory.hpp"
+#include "util.h" // xyz2llh
```

Comment:
Could you take this opportunity to move xyz2llh into a separate hpp file? Adding util to trajectory will add a lot of unwanted includes, also to other files #including this one.